superset-frontend/src/SqlLab/components/SqlEditor/index.tsx

```diff
@@ -847,7 +859,17 @@ const SqlEditor: React.FC<Props> = ({
           )}
         </ResizableSidebar>
       </CSSTransition>
-      {showEmptyState ? (
+      {shouldLoadQueryEditor ? (
+        <div
+          data-test="sqlEditor-loading"
+          css={css`
+            flex: 1;
+            padding: ${theme.gridUnit * 4}px;
+          `}
+        >
+          <Skeleton active />
```

Comment:
Given that the `Skeleton` refers to the whole query editor, could you make the spinner centralized? Right now, it's aligned to the left.

Reply:
The skeleton appears visually better when aligned to the left. (By the way, the attached video used a different loading indicator.)

Reply:
![_DEV__Superset](https://github.com/apache/superset/assets/1392866/9f96d160-3c42-4581-9452-15d4365a832f)